vainfo/vainfo.c

```diff
@@ -93,7 +93,7 @@ int main(int argc, const char* argv[])
   const char *name = strrchr(argv[0], '/'); 
   VAProfile profile, *profile_list = NULL;
   int num_profiles, max_num_profiles, i;
-  VAEntrypoint entrypoint, entrypoints[10];
+  VAEntrypoint entrypoint, entrypoints[11];
```

Comment:
The number of the defined entrypoints in va.h is 12 now although 11 is safe for existing vaapi drivers.  In addition, we can use vaMaxNumEntrypoints() to get the maximum number of entrypoints supported by the underlying driver, then allocate the memory for entrypoint list per the returned value.  If do so, we needn't change it again for new profiles added in the future.